### DISCUSSION TITLE
i cant set up

### DISCUSSION THREAD

**Discussion by tewisme** [cpeditor/cpeditor#1197] [Q&A]
I know cp from some friend
I download it on web but after dowload file set up, i cant run that file to set up
anyone know why?

**Comment by ouuan** [comment]
Well, you almost provided no information. Perhaps it would be helpful to read [How To Ask Questions The Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html).

You at least need to explain what "can't run" exactly means. Additionally, you may provide the exact name of the file you download, your operating system, and maybe some screenshots.
